packages/server/test/fastify.spec.ts

```diff
@@ -35,6 +35,9 @@ describe('Fastify', () => {
       url: '/mypath',
       method: ['GET', 'POST', 'OPTIONS'],
       handler: async (req, reply) => {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore Types of property 'req' are incompatible.
+        // Type 'IncomingMessage' is missing the following properties from type 'Http2ServerRequest': authority, scheme, streamts(2345)
```

Comment:
That's why we shouldn't use IncomingMessage or Http2ServerRequest interfaces directly.

Reply:
it is falling through from `const rawRequest = nodeRequest.raw || nodeRequest.req || nodeRequest;`